Compentio.SourceMapper.Tests/Metadata/PropertyMetadataTestBase.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection;
+
+namespace Compentio.SourceMapper.Tests.Metadata
+{
+    public abstract class PropertyMetadataTestBase
+    {
+        protected abstract string FakeNamespace { get; }
```

Comment:
Lets use everywhere instead of Fake -> Mock

Reply:
Fixed.